conans/client/command.py

```diff
@@ -904,6 +899,27 @@ def run(self, *args):
         return errors
 
 
+def get_reference_fields(arg_reference):
+    """
+    :param arg_reference: String with a complete reference, or only user/channel, or only user
+                          (channel testing became default)
+    :return: name, version, user and channel, in a tuple
+    """
+    try:
+       name_version, user_channel = arg_reference.split("@")
+       name, version = name_version.split("/")
+       user, channel = user_channel.split("/")
+    except:
+       name, version = None, None
+       try:
+            user, channel = arg_reference.split("/")
+       except:
+            user = arg_reference
+            channel = "testing"
```

Comment:
The removal of ``testing`` default in ``conan create`` was intentional. I feel it is confusing ``conan create Pkg/0.1@user`` and it will use by default the ``testing`` channel. I would try to deprecate that default and enforce the definition of channel, always.